test/unit/network_spec.js

```diff
@@ -94,7 +94,7 @@ describe('network', function() {
         rangeChunkSize: 65536,
         disableStream: false,
       },
-      disableRange: false
+      disableRange: true
```

Comment:
Well, I suppose that you're correct about this being unrelated!
Since I needed to make a couple of adjustments in this file anyway, I figured that it couldn't hurt to clean up a couple of things. But your comment makes me realize that I may have gone overboard here, so I'll revert this to the essential changes only. Thank you!